pysindy/optimizers/constrained_sr3.py

```diff
@@ -90,6 +91,10 @@ class ConstrainedSR3(SR3):
         the regressors X will be normalized before regression by subtracting
         the mean and dividing by the l2-norm.
 
+    normalize_columns : boolean, optional (default False)
+        Normalize the columns of x (the SINDy library terms). Note that
+        this parameter is incompatible with the constraints!
```

Comment:
Do we validate this at all?

Reply:
no, it would be good to write a test in test_optimizers that tests this routine for each one.

Reply:
I'll do this today.